### PR TITLE
Bring version number current in README

### PR DESCRIPTION
**Pull request by gliptak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 This repository contains the build script and `Dockerfile` used to produce the
-binaries for each restic release starting at 0.6.1.
+binaries for each restic release starting at 0.9.2.
```
Comment (fd0) [inline README.md]: Hm, that's not correct, can you revert this change?

**Review by fd0 — CHANGES_REQUESTED** [review]
Thanks a lot for improving our documentation! Can you please revert the first occurrence of the version? Then we're good to merge this!

**Comment by gliptak** [comment]
updated

**Review by fd0 — APPROVED** [review]
Cool, thanks!

**Pull request merged**